internal/test/wmcb/wmcb_test.go

```diff
@@ -88,9 +90,38 @@ type pkgInfo struct {
 	shaType string
 }
 
+type wmcbFramework struct {
+	// TestFramework holds the instantiation of test suite being executed
+	*e2ef.TestFramework
+	// latestCniPluginsVersion is the latest 0.8.x version of CNI Plugins
+	LatestCniPluginsVersion string
```

Comment:
why is this a public field?

Reply:
Moved it to framework.go as it is used both in wmcb_test.go and wsu_test.go